libr/main/rafind2.c

```diff
@@ -375,22 +376,40 @@ R_API int r_main_rafind2(int argc, const char **argv) {
 		case 'b':
 			bsize = r_num_math (NULL, opt.arg);
 			break;
-		case 'x':
-			mode = R_SEARCH_KEYWORD;
-			hexstr = 1;
-			widestr = 0;
-			r_list_append (keywords, (void*)opt.arg);
-			break;
 		case 'M':
 			// XXX should be from hexbin
 			mask = opt.arg;
 			break;
 		case 'f':
 			from = r_num_math (NULL, opt.arg);
 			break;
+		case 'F':
+			{
+				size_t data_size;
+				char *data = r_file_slurp (opt.arg, &data_size);
+				if (!data) {
+					eprintf ("Cannot slurp '%s'\n", opt.arg);
+					return 1;
+				}
+				char *hexdata = r_hex_bin2strdup ((ut8*)data, data_size);
```

Comment:
This hex conversion is of course a lot of unnecessary overhead. Do you plan to get rid of this in future prs too?

Reply:
Yes, this is exactly what i was refering to, because the rest of keywords are using string representations and appending strings in a list instead of appending the keyword instance itself. I will change this in future commits, this will also save the memleak

Reply:
good, then lgtm.